src/core.jl

```diff
@@ -144,6 +144,10 @@ nrows(y::AbstractVector) = length(y)
 
 reformat(X) = reformat(X, scitype(X))
 
+# ---------------------------------
+# Reformatting matrices
+reformat(X, ::Type{<:AbstractMatrix}) = permutedims(X)
```

Comment:
Yes, when MLJ users provide inputs in the form of a matrix, the observations are rows, which is the opposite of what the internal Flux operations are expecting. 

However, I wouldn't we be better off with and adjoint here, instead of `permute dims`? Then, if a user provides an adjoint, the adjoints cancel and there is no performance cost at all. No?

Reply:
Yes, good point! Have changed that